detection/persistence/unexpected-chrome-extensions.sql

```diff
@@ -339,7 +339,10 @@ WHERE
     'true,Yuri Konotopov <ykonotopov@gnome.org>,GNOME Shell integration,gphhapmejobijbbhgpjhcjognlahblep',
     'true,,Zoom,hmbjbjdpkobdjplfobhljndfdfdipjhg',
     'true,,ZoomInfo Engage Chrome Extension,mnbjlpbmllanehlpbgilmbjgocpmcijp',
-    'true,,Zoom Scheduler,kgjfgplpablkjnlkjmjdecgdpfankdle'
+    'true,,Zoom Scheduler,kgjfgplpablkjnlkjmjdecgdpfankdle',
+    'true,AgileBits,1Password \xE2\x80\x93 Password Manager,aeblfdkhhhdcdjpifhhbdiojplfjncoa',
```

Comment:
The em dash exceptions are already present further up in the list and we were still getting alerts for this. I think we may need the hex variants which I've already applied in Kolide.